zipline/algorithm.py

```diff
@@ -324,10 +321,8 @@ def __init__(self, *args, **kwargs):
             self.blotter = Blotter(
                 data_frequency=self.data_frequency,
                 asset_finder=self.asset_finder,
-                slippage_func=VolumeShareSlippage(),
-                commission=PerShare(),
```

Comment:
Instead of adding in the two new future parameters here I just removed these two arguments because they were passing the same values as the defaults anyway.